dotcom-rendering/src/web/layouts/StandardLayout.tsx

```diff
@@ -485,6 +490,18 @@ export const StandardLayout = ({ CAPI, NAV, format, palette }: Props) => {
 									)}
 							</div>
 						</GridItem>
+						<GridItem area="matchtabs" element="aside">
+							<div css={maxWidth}>
+								{format.design === ArticleDesign.DeadBlog ||
+									(format.design === ArticleDesign.LiveBlog &&
+										CAPI.matchUrl && (
+											<Placeholder
+												rootId="match-tabs"
+												height={40}
+											/>
+										))}
+							</div>
+						</GridItem>
```

Comment:
```suggestion
						<GridItem area="matchtabs" element="aside">
							<div css={maxWidth}>
								{CAPI.matchUrl && (
									<Placeholder
										rootId="match-tabs"
										height={40}
									/>
								)}
							</div>
						</GridItem>
```

Reply:
I missed this but we don't need this format check here as we're never going to be a live blog at this point, not in StandardLayout